content/components/icon.mdx

```diff
@@ -0,0 +1,216 @@
+    />
+    <Caption>Don't resize icons.</Caption>
+  </Dont>
+</DoDontContainer>
+
+## System vs user-selectable
+
+There are two types of icons: `system icons` and `user-selectable` icons.
+
+System icons are used to represent core actions and concepts within the user interface. These icons are typically used consistently throughout the product and are an important part of our visual language. Examples of system icons might include a [trash](/octicons/trash-16) icon for deleting items, or a [plus-circle](/octicons/plus-circle-16) icon for adding new items.
+
+User-selectable icons, on the other hand, are used to allow users to customize their experience or make a personal connection with the product. User-selectable icons are typically less central to the functionality of the user interface and may be changed by the user as desired.
+
+At GitHub we currently use Emojies as user-selectable icons as they are widely recognized and understood, making them an effective way to convey emotions or personality. One benefit of using emojis as user-selectable icons is that they are visually distinct from the system icons used in the design system. This helps users to easily understand that the emojis are optional and can be selected by the user, rather than being a part of the core functionality of the user interface.
+
+Allowing users to pick [Octicons](/octicons/) (system icons) is strongly discouraged and could create a confusing experience when they are used in the wrong context. For example a user could pick a [issue-closed](https://primer.style/octicons/issue-closed-16) icon and use the label `In progress`.
+
+<DoDontContainer>
+  <Do>
+    <img
+      src="https://user-images.githubusercontent.com/980622/209167301-278f8dff-70a1-4bc3-9ad9-335fb365bd67.png"
+      role="presentation"
+      width="456"
+      alt="A navigation side bar with multiple items including 'To do', 'Learning' and ''Ready to go. Each item has a emoji assigned and is displayed on the left of the text."
+    />
+    <Caption>Use Emojies as user-selectable icons.</Caption>
+  </Do>
+  <Dont>
+    <img
+      src="https://user-images.githubusercontent.com/980622/209167308-0c8cca85-5d78-4c60-b69b-13f5afbf0254.png"
+      role="presentation"
+      width="456"
+      alt="A navigation side bar with multiple items including 'To do', 'Learning' and ''Ready to go. Each item has a octicon assigned in a different icon and is displayed on the left of the text."
+    />
+    <Caption>Don't use Octicons for user-selectable icons.</Caption>
+  </Dont>
+</DoDontContainer>
+
+Note that internally we had multiple discussions about introducing user-selectable octicons but no specific work or explorations have been started on this yet. This work initially would involve creating a set of octicons that look visually different from our existing style.
```

Comment:
I have mixed feelings about this part. Even if this is true, should we add a note like that? This might sound like emojis are a temporary/quick solution 🤔.

```suggestion
Note that internally we had multiple discussions about introducing user-selectable octicons but no specific work or explorations have been started on this yet. This work initially would involve creating a set of octicons that look visually different from our existing style.
```

Reply:
You're right. Removed it for now.